File: 250317/좀 더 어려운 수학 점수/math-scores-are-more-difficult.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int a_math = 0;
    int a_eng = 0;
    int b_math = 0;
    int b_eng = 0;

    cin >> a_math >> a_eng >> b_math >> b_eng;

    if (a_math > b_math)
    {
        cout << "A";
    }
    else if (a_math < b_math)
    {
        cout << "B";
    }
    else if (a_math == b_math && a_eng > b_eng)
    {
        cout << "A";
    }
    else if (a_math == b_math && a_eng < b_eng)
    {
        cout << "B";
    }
    return 0;
}

File: 250317/좀 더 어려운 수학 점수/math-scores-are-more-difficult.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a_math = sc.nextInt();
        int a_eng = sc.nextInt();
        int b_math = sc.nextInt();
        int b_eng = sc.nextInt();

        if(a_math > b_math) {
            System.out.println("A");
        }
        else if (a_math < b_math) {
            System.out.println("B");
        }
        else if(a_math == b_math && a_eng > b_eng) {
            System.out.println("A");
        }
        else if(a_math == b_math && a_eng < b_eng) {
            System.out.println("B");
        }
        sc.close();
    }
}